src/components/ReportTransaction.js

```diff
@@ -1,10 +1,19 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import {Text, View, Pressable} from 'react-native';
+import lodashGet from 'lodash/get';
+import {
+    Text, View, Pressable, ActivityIndicator,
+} from 'react-native';
+import {
+    withOnyx,
+} from 'react-native-onyx';
```

Comment:
Can you make both of these imports one-liners?

Reply:
Made withOnyx one-liner.
But, eslint doesn't allow `import {Text, View, Pressable, ActivityIndicator,} from 'react-native';`
to be a one-liner.

Reply:
Can you try without the final comma after ActivityIndicator? e.g.: `import {Text, View, Pressable, ActivityIndicator} from 'react-native';`

Reply:
Nevermind, I see what you mean - I can't figure out why eslint is picky about that line, but oh well :D